spring-remoting/remoting-http/server/pom.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Umh... actually not.  Can this be of any help ? <https://github.com/tesla/m2eclipse-mavenarchiver/issues/9> 

Or... did you try to compile it from the command line ? Here's what it run on my machine:

	C:\Users\danidemi\git\tutorials\spring-remoting>dir
	 Il volume nell'unità C non ha etichetta.
	 Numero di serie del volume: C2AD-2B80

	 Directory di C:\Users\danidemi\git\tutorials\spring-remoting

	09/01/2017  22:34    <DIR>          .
	09/01/2017  22:34    <DIR>          ..
	09/01/2017  22:37    <DIR>          .idea
	09/01/2017  22:34             4.157 pom.xml
	09/01/2017  22:34    <DIR>          remoting-http
	01/01/2017  19:04               606 remoting.iml
				   2 File          4.763 byte
				   4 Directory  1.265.322.749.952 byte disponibili

	C:\Users\danidemi\git\tutorials\spring-remoting>mvn install
	[INFO] Scanning for projects...
	[INFO] ------------------------------------------------------------------------
	[INFO] Reactor Build Order:
	[INFO]
	[INFO] spring-remoting
	[INFO] spring-remoting-http
	[INFO] spring-remoting-http-api
	[INFO] spring-remoting-http-server
	[INFO] spring-remoting-http-client
	[INFO]
	[INFO] ------------------------------------------------------------------------
	[INFO] Building spring-remoting 1.0-SNAPSHOT
	[INFO] ------------------------------------------------------------------------
	[INFO]
	[INFO] --- maven-install-plugin:2.4:install (default-install) @ spring-remoting ---
	[INFO] Installing C:\Users\danidemi\git\tutorials\spring-remoting\pom.xml to C:\Users\danidemi\.m2\repository\com\baeldung\spring-remoting\1.0-SNAPSHOT\spring-remoting-1.0-SNAPSHOT.pom
	[INFO]
	[INFO] ------------------------------------------------------------------------
	[INFO] Building spring-remoting-http 1.0-SNAPSHOT
	[INFO] ------------------------------------------------------------------------
	[INFO]
	[INFO] --- maven-install-plugin:2.4:install (default-install) @ spring-remoting-http ---
	[INFO] Installing C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\pom.xml to C:\Users\danidemi\.m2\repository\com\baeldung\spring-remoting-http\1.0-SNAPSHOT\spring-remoting-http-1.0-SNAPSHOT.pom
	[INFO]
	[INFO] ------------------------------------------------------------------------
	[INFO] Building spring-remoting-http-api 1.0-SNAPSHOT
	[INFO] ------------------------------------------------------------------------
	[INFO]
	[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ api ---
	[WARNING] Using platform encoding (Cp1252 actually) to copy filtered resources, i.e. build is platform dependent!
	[INFO] Copying 0 resource
	[INFO]
	[INFO] --- maven-compiler-plugin:3.6.0:compile (default-compile) @ api ---
	[INFO] Changes detected - recompiling the module!
	[INFO] Compiling 4 source files to C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\api\target\classes
	[INFO]
	[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ api ---
	[WARNING] Using platform encoding (Cp1252 actually) to copy filtered resources, i.e. build is platform dependent!
	[INFO] skip non existing resourceDirectory C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\api\src\test\resources
	[INFO]
	[INFO] --- maven-compiler-plugin:3.6.0:testCompile (default-testCompile) @ api ---
	[INFO] Nothing to compile - all classes are up to date
	[INFO]
	[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ api ---
	[INFO] No tests to run.
	[INFO]
	[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ api ---
	[INFO] Building jar: C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\api\target\api-1.0-SNAPSHOT.jar
	[INFO]
	[INFO] --- maven-install-plugin:2.4:install (default-install) @ api ---
	[INFO] Installing C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\api\target\api-1.0-SNAPSHOT.jar to C:\Users\danidemi\.m2\repository\com\baeldung\api\1.0-SNAPSHOT\api-1.0-SNAPSHOT.jar
	[INFO] Installing C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\api\pom.xml to C:\Users\danidemi\.m2\repository\com\baeldung\api\1.0-SNAPSHOT\api-1.0-SNAPSHOT.pom
	[INFO]
	[INFO] ------------------------------------------------------------------------
	[INFO] Building spring-remoting-http-server 1.0-SNAPSHOT
	[INFO] ------------------------------------------------------------------------
	[INFO]
	[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ spring-remoting-http-server ---
	[WARNING] Using platform encoding (Cp1252 actually) to copy filtered resources, i.e. build is platform dependent!
	[INFO] Copying 1 resource
	[INFO]
	[INFO] --- maven-compiler-plugin:3.6.0:compile (default-compile) @ spring-remoting-http-server ---
	[INFO] Changes detected - recompiling the module!
	[INFO] Compiling 4 source files to C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\server\target\classes
	[INFO]
	[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ spring-remoting-http-server ---
	[WARNING] Using platform encoding (Cp1252 actually) to copy filtered resources, i.e. build is platform dependent!
	[INFO] skip non existing resourceDirectory C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\server\src\test\resources
	[INFO]
	[INFO] --- maven-compiler-plugin:3.6.0:testCompile (default-testCompile) @ spring-remoting-http-server ---
	[INFO] No sources to compile
	[INFO]
	[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ spring-remoting-http-server ---
	[INFO] No tests to run.
	[INFO]
	[INFO] --- maven-war-plugin:3.0.0:war (default-war) @ spring-remoting-http-server ---
	[INFO] Packaging webapp
	[INFO] Assembling webapp [spring-remoting-http-server] in [C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\server\target\spring-remoting-http-server-1.0-SNAPSHOT]
	[INFO] Processing war project
	[INFO] Copying webapp resources [C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\server\src\main\webapp]
	[INFO] Webapp assembled in [287 msecs]
	[INFO] Building war: C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\server\target\spring-remoting-http-server-1.0-SNAPSHOT.war
	[INFO]
	[INFO] --- maven-install-plugin:2.4:install (default-install) @ spring-remoting-http-server ---
	[INFO] Installing C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\server\target\spring-remoting-http-server-1.0-SNAPSHOT.war to C:\Users\danidemi\.m2\repository\com\baeldung\spring-remoting-http-server\1.0-SNAPSHOT\spring-remoting-http-server-1.0-
	SNAPSHOT.war
	[INFO] Installing C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\server\pom.xml to C:\Users\danidemi\.m2\repository\com\baeldung\spring-remoting-http-server\1.0-SNAPSHOT\spring-remoting-http-server-1.0-SNAPSHOT.pom
	[INFO]
	[INFO] ------------------------------------------------------------------------
	[INFO] Building spring-remoting-http-client 1.0-SNAPSHOT
	[INFO] ------------------------------------------------------------------------
	[INFO]
	[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ spring-remoting-http-client ---
	[WARNING] Using platform encoding (Cp1252 actually) to copy filtered resources, i.e. build is platform dependent!
	[INFO] skip non existing resourceDirectory C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\client\src\main\resources
	[INFO]
	[INFO] --- maven-compiler-plugin:3.6.0:compile (default-compile) @ spring-remoting-http-client ---
	[INFO] Changes detected - recompiling the module!
	[INFO] Compiling 2 source files to C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\client\target\classes
	[INFO]
	[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ spring-remoting-http-client ---
	[WARNING] Using platform encoding (Cp1252 actually) to copy filtered resources, i.e. build is platform dependent!
	[INFO] skip non existing resourceDirectory C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\client\src\test\resources
	[INFO]
	[INFO] --- maven-compiler-plugin:3.6.0:testCompile (default-testCompile) @ spring-remoting-http-client ---
	[INFO] No sources to compile
	[INFO]
	[INFO] --- maven-surefire-plugin:2.12.4:test (default-test) @ spring-remoting-http-client ---
	[INFO] No tests to run.
	[INFO]
	[INFO] --- maven-jar-plugin:2.4:jar (default-jar) @ spring-remoting-http-client ---
	[INFO] Building jar: C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\client\target\spring-remoting-http-client-1.0-SNAPSHOT.jar
	[INFO]
	[INFO] --- maven-install-plugin:2.4:install (default-install) @ spring-remoting-http-client ---
	[INFO] Installing C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\client\target\spring-remoting-http-client-1.0-SNAPSHOT.jar to C:\Users\danidemi\.m2\repository\com\baeldung\spring-remoting-http-client\1.0-SNAPSHOT\spring-remoting-http-client-1.0-
	SNAPSHOT.jar
	[INFO] Installing C:\Users\danidemi\git\tutorials\spring-remoting\remoting-http\client\pom.xml to C:\Users\danidemi\.m2\repository\com\baeldung\spring-remoting-http-client\1.0-SNAPSHOT\spring-remoting-http-client-1.0-SNAPSHOT.pom
	[INFO] ------------------------------------------------------------------------
	[INFO] Reactor Summary:
	[INFO]
	[INFO] spring-remoting .................................... SUCCESS [  1.653 s]
	[INFO] spring-remoting-http ............................... SUCCESS [  0.111 s]
	[INFO] spring-remoting-http-api ........................... SUCCESS [  8.139 s]
	[INFO] spring-remoting-http-server ........................ SUCCESS [  2.372 s]
	[INFO] spring-remoting-http-client ........................ SUCCESS [  0.233 s]
	[INFO] ------------------------------------------------------------------------
	[INFO] BUILD SUCCESS
	[INFO] ------------------------------------------------------------------------
	[INFO] Total time: 12.612 s
	[INFO] Finished at: 2017-01-09T22:37:50+01:00
	[INFO] Final Memory: 23M/291M
	[INFO] ------------------------------------------------------------------------